h2integrate/control/control_rules/pyomo_control_options.py

```diff
@@ -0,0 +1,148 @@
+
+            - **grid_charging** (bool, default=True): Can the battery charge from the grid.
+
+            - **pv_charging_only** (bool, default=False): Whether restricted to only charge from PV
+            (ITC qualification).
+
+            - **include_lifecycle_count** (bool, default=True): Should battery lifecycle counting
+            be included.
+
+            - **lifecycle_cost_per_kWh_cycle** (float, default=0.0265): If include_lifecycle_count,
+            cost per kWh cycle.
+
+            - **max_lifecycle_per_day** (int, default=None): If include_lifecycle_count, how many
+            cycles allowed per day.
+
+            - **n_look_ahead_periods** (int, default=48): Number of time periods dispatch
+            looks ahead.
+
+            - **n_roll_periods** (int, default=24): Number of time periods simulation rolls forward
+            after each dispatch.
+
+            - **time_weighting_factor** (float, default=0.995): Discount factor for the time periods
+            in the look ahead period.
+
+            - **log_name** (str, default=''): Dispatch log file name, empty str will result in no
+            log (for development).
+
+            - **is_test_start_year** (bool, default=False): If True, simulation solves for first 5
+            days of the year.
+
+            - **is_test_end_year** (bool, default=False): If True, simulation solves for last 5
+            days of the year.
+
+            - **use_clustering** (bool, default=False): If True, the simulation will be run for a
+            selected set of "exemplar" days.
+
+            - **n_clusters** (int, default=30).
+
+            - **clustering_weights** (dict, default={}): Custom weights used for classification
+            metrics for data clustering. If empty, default weights will be used.
+
+            - **clustering_divisions** (dict, default={}): Custom number of averaging periods for
+            classification metrics for data clustering. If empty, default values will be used.
+
+            - **use_higher_hours** bool (default = False): if True, the simulation will run extra
+            hours analysis (must be used with load following)
+
+            - **higher_hours** (dict, default = {}): Higher hour count parameters: the value of
+            power that must be available above the schedule and the number of hours in a row
+
+    """
+
+    def __init__(self, control_options: dict | None = None):
+        self.solver: str = "cbc"
+        self.solver_options: dict = {}  # used to update solver options, see solver for options
+        self.battery_dispatch: str = "simple"
+        self.include_lifecycle_count: bool = True
+        self.lifecycle_cost_per_kWh_cycle: float = (
+            0.0265  # Estimated using SAM output (lithium-ion battery)
+        )
+        self.max_lifecycle_per_day: int = np.inf
+        self.grid_charging: bool = True
+        self.pv_charging_only: bool = False
```

Comment:
Some of these might be outdated or not used? Check to see if we can remove any imo